### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by TysonAndre**
Start setting up (failing) eslint rules

Fix some occurrences of unused variables and harmless uses of
hasOwnProperty (in hardcoded schema)

Set up github workflows

Drop use of bluebird because Node.js has efficient native Promises.

Update CHANGELOG.md

Hold back jsen dependency. See https://github.com/ifwe/monocle-api/issues/86

### CODE REVIEW

File: lib/Route.js
Comment (erikjohannessen) [inline lib/Route.js]: This is to help debugging for now?

File: CHANGELOG.md
Comment (erikjohannessen) [inline CHANGELOG.md:5]: why go to 4.0.0 now, if we're holding off on the breaking change?

**Review by erikjohannessen — APPROVED** [review]

File: lib/Route.js
Comment (TysonAndre) [reply lib/Route.js]: Reverted, I was looking into the failure from updating the jsen patch version

File: CHANGELOG.md
Comment (TysonAndre) [reply CHANGELOG.md:5]: Fixed, any applications that had failures due to using older Node.js versions should be pretty obvious, so a major version bump shouldn't be needed for that. The package.json "engines" change added after this also makes this obvious

**Comment by TysonAndre** [comment]
No release of `jsen` fixes the audit warning, if I remember correctly, the maintainers of `jsen` maintain that only trusted schemas should be used for schema validation, which is true for uses of monocle-api as well.

I haven't looked into the impact of switching to `ajv` but smaller changes would be easier to validate

**Comment by erikjohannessen** [comment]
lgtm

**Pull request merged**
